boards/arm/mimxrt685_evk/pinmux.c

```diff
@@ -444,7 +444,7 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 #endif
 
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay) && CONFIG_FLASH
-const uint32_t port1_pin11_config = (/* Pin is configured as FLEXSPI0B_DATA0 */
+	uint32_t port1_pin11_config = (/* Pin is configured as FLEXSPI0B_DATA0 */
```

Comment:
Please move all the whitespace fixes into a separate commit

Reply:
I will make a separate PR

Reply:
PR https://github.com/zephyrproject-rtos/zephyr/pull/36785 created to address the other issues with this file.